packages/esm-ward-app/src/ward-patient-card/card-rows/mother-child-row.scss

```diff
@@ -5,8 +5,11 @@
   display: flex;
 }
 
-.motherOrBabyIconDiv {
+.motherOrBabyRowElementsDev {
```

Comment:
typo, "dev" when I assume you mean "div"?

Reply:
Yes... thanks for catching